Example/Example/SomeView.swift

```diff
@@ -16,7 +17,7 @@ struct SomeView: View {
         Text("Hello from SwiftUI 🙃")
             .onAppear {
                 span.end()
-            }
+            }.bugsnagTraced("My text view")
```

Comment:
Can we show how you could instrument the whole screen?  As currently setup I think this means adding `.bugsnagTraced` to `SomeView()` in `ViewController.showSwiftUIView()`